Sources_v3/StreamChatUI/ChatChannel/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatFileAttachmentListView {
+    open class ItemView: View, UIConfigProvider {
+        public var attachment: _ChatMessageAttachment<ExtraData>? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        public var tapHandler: () -> Void = {}
+
+        public var actionIconSize = CGSize(width: 20, height: 20)
+
+        // MARK: - Subviews
+
+        public private(set) lazy var fileIconImageView: UIImageView = {
+            let imageView = UIImageView()
+            imageView.contentMode = .scaleAspectFit
+            return imageView.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileNameLabel: UILabel = {
+            let label = UILabel()
+            label.font = UIFont.preferredFont(forTextStyle: .body).bold
+            label.adjustsFontForContentSizeCategory = true
+            return label.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileSizeLabel: UILabel = {
+            let label = UILabel()
+            label.font = UIFont.preferredFont(forTextStyle: .footnote)
+            label.adjustsFontForContentSizeCategory = true
+            return label.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var activityIndicator: UIActivityIndicatorView = {
+            let indicator = UIActivityIndicatorView()
+            indicator.hidesWhenStopped = true
+            return indicator.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var actionIconImageView: UIImageView = {
+            let imageView = UIImageView()
+            imageView.contentMode = .scaleAspectFit
+            return imageView.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var spinnerAndSizeStack: UIStackView = {
+            let stack = UIStackView()
+            stack.axis = .horizontal
+            return stack.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileNameAndSizeStack: UIStackView = {
+            let stack = UIStackView()
+            stack.axis = .vertical
+            stack.alignment = .leading
```

Comment:
How you decide what going in `defaultAppearance` and what stay inside lazy?
Why `alignment` and `axis` are part of lazy init, but `spacing` is part of appearance (not even layout)?

Reply:
No clue 🤷‍♂️